turbo.json

```diff
@@ -17,6 +17,17 @@
         "index.js"
       ]
     },
+    "@skillrecordings/config#build": {
+      "dependsOn": [
+        "^build",
+        "$CONVERTKIT_API_SECRET",
+        "$NEXT_PUBLIC_CONVERTKIT_TOKEN",
+        "$NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM"
+      ],
```

Comment:
this is something that I need to spend more time with to understand more completely. Generally I think we need to remove all of these vars from ALL packages and instead pass them in to the package as config from the apps

https://turborepo.org/docs/core-concepts/caching#alter-caching-based-on-environment-variables-and-files

that's the advice from the docs and it makes sense